### DISCUSSION TITLE
Could not find model for point [ 0.529087 3.40752 -11.9262 ] in group 5

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#713] [Q&A]
I find a `Nastran format` model , I want to use a `point source` to test if it can use SeisSol's code to calculate this model, it is able to convert to `.msh` and `.h5` and `.xdmf` formats, but I run into this problem when I run it, do you know this What is the reason? The files I used are in the link below,the `Nastran file` is too big,so I split it into two parts. and I save it as `.msh file`，it  also in the below link:
https://github.com/buzhiqiuyuebai/submarine

![image](https://user-images.githubusercontent.com/114474894/198008424-7a2b722b-2155-4857-8e12-6ae276469722.png)

**Comment by Thomas-Ulrich** [comment]
https://github.com/SeisSol/Examples/blob/master/WP2_LOH3/loh3.yaml has 2 
layers, tagged 1 and 2
Your mesh has a region tagged 5.
That's why easi is not able to find material values for it (given 5 is 
filtered out from the group filters 1 and 2).

**Reply by buzhiqiuyuebai** [reply]
> https://github.com/SeisSol/Examples/blob/master/WP2_LOH3/loh3.yaml has 2 layers, tagged 1 and 2 Your mesh has a region tagged 5. That's why easi is not able to find material values for it (given 5 is filtered out from the group filters 1 and 2).
> […](#)
> On 26.10.22 13:47, buzhiqiuyuebai wrote: I find a |Nastran format| model , I want to use a |point source| to test if it can use SeisSol's code to calculate this model, it is able to convert to |.msh| and |.h5| and |.xdmf| formats, but I run into this problem when I run it, do you know this What is the reason? The files I used are in the link below,the |Nastran file| is too big,so I split it into two parts. and I save it as |.msh file|，it also in the below link: https://github.com/buzhiqiuyuebai/submarine image <https://user-images.githubusercontent.com/114474894/198008424-7a2b722b-2155-4857-8e12-6ae276469722.png> — Reply to this email directly, view it on GitHub <#713>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADHI5DVXEXGV7AO22FYKMYDWFEK6DANCNFSM6AAAAAARO5C3KU>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

I will have a try, Thank you foy your answer.

**Reply by buzhiqiuyuebai** [reply]
> https://github.com/SeisSol/Examples/blob/master/WP2_LOH3/loh3.yaml has 2 layers, tagged 1 and 2 Your mesh has a region tagged 5. That's why easi is not able to find material values for it (given 5 is filtered out from the group filters 1 and 2).
> […](#)
> On 26.10.22 13:47, buzhiqiuyuebai wrote: I find a |Nastran format| model , I want to use a |point source| to test if it can use SeisSol's code to calculate this model, it is able to convert to |.msh| and |.h5| and |.xdmf| formats, but I run into this problem when I run it, do you know this What is the reason? The files I used are in the link below,the |Nastran file| is too big,so I split it into two parts. and I save it as |.msh file|，it also in the below link: https://github.com/buzhiqiuyuebai/submarine image <https://user-images.githubusercontent.com/114474894/198008424-7a2b722b-2155-4857-8e12-6ae276469722.png> — Reply to this email directly, view it on GitHub <#713>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADHI5DVXEXGV7AO22FYKMYDWFEK6DANCNFSM6AAAAAARO5C3KU>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

I accept your advice, and I can solve that problem, but I encounter  a new problem, that is 
![image](https://user-images.githubusercontent.com/114474894/198218619-7406c653-428c-410e-9f11-ecb3ba684046.png)
I checked related problems on the Internet, and some people said that it was caused by openmpi, but I can run the examples you provided, such as WP_LOH3, so I don't think this is the reason. Have you encountered this problem?

**Reply by Thomas-Ulrich** [reply]
I would double check the boundary conditions, as there is no error check for that.
You can use this script:
https://github.com/SeisSol/Meshing/tree/master/vizualizeBoundaryConditions

**Reply by Thomas-Ulrich** [reply]
note that we never use order 2.

**Reply by buzhiqiuyuebai** [reply]
> I would double check the boundary conditions, as there is no error check for that. You can use this script: https://github.com/SeisSol/Meshing/tree/master/vizualizeBoundaryConditions

I use order 6,but it also has problem, and Could you please tell me how this script can be used to check the mesh quality.
![image](https://user-images.githubusercontent.com/114474894/198294793-7bb5383a-02aa-42ff-a2ac-127e6aa61a63.png)

**Reply by Thomas-Ulrich** [reply]
mesh quality: the minimum time step and the LTS speed up are good hints of mesh quality.
if dt is very small, or if the speed up is huge (> 50) then the mesh is of bad quality.
Also in pumgen, you can see the minimum insphere size.
If much smaller than 1/10 of your mesh minimum mesh size, then it is ok bad quality.
